src/components/Post/MoreText.tsx

```diff
@@ -0,0 +1,28 @@
+import { Collapse, CollapseProps, Button } from '@chakra-ui/react';
+import { useState } from 'react';
+
+type MoreTextProps = CollapseProps & {
+  text: string;
+  maxHeight: number;
+};
+
+const MoreText = ({ text, maxHeight, ...props }: MoreTextProps) => {
+  const [toggle, setToggle] = useState(false);
+
+  const handleToggle = () => {
+    setToggle((prev) => !prev);
+  };
+
+  return (
+    <>
+      <Collapse {...props} startingHeight={maxHeight} in={toggle}>
+        {text}
+      </Collapse>
+      <Button size="sm" onClick={handleToggle} mt="1rem">
```

Comment:
```suggestion
      <Button size="sm" onClick={handleToggle} mt={4}>
```
chakra에서 정의된 값으로 바꿀 수 있을 것 같습니다!!